packages/cozy-stack-client/src/NextcloudFilesCollection.js

```diff
@@ -0,0 +1,46 @@
+import DocumentCollection from './DocumentCollection'
+
+const NEXTCLOUD_FILES_DOCTYPE = 'io.cozy.remote.nextcloud.files'
+
+const normalizeDoc = DocumentCollection.normalizeDoctypeJsonApi(
+  NEXTCLOUD_FILES_DOCTYPE
+)
+const normalizeNextcloudFile = (sourceAccount, path) => file => {
+  const extendedAttributes = {
+    ...file.attributes,
+    path: `${path}${path.endsWith('/') ? '' : '/'}${file.attributes.name}`,
+    cozyMetadata: {
+      ...file.attributes.cozyMetadata,
+      sourceAccount
```

Comment:
Why do we have to manually set the `sourceAccount`? I feel like this should be up to the client to set it, just like the other `cozyMetadata` attributes?

Reply:
This is for convenience, because in order to use all the routes in the stack we need to have the Nextcloud path and sourceAccount pair. This is only present in the shortcut describing the Nextcloud. To reuse the same action as the io.cozy.files, I've introduced it into the file so that you only have to pass a file as a parameter as expected